app/helpers/text_scanner.rb

```diff
@@ -21,10 +21,13 @@ def self.scan_with_gcv(image_path)
     self.retrieve_key_file
 
     require 'google/cloud/vision'
-    vision = Google::Cloud::Vision.new(project: ENV['GOOGLE_PROJECT_ID'])
-    image = vision.image(image_path)
-    image.context.languages = [:en]
-    image.document.text
+    image_annotator = Google::Cloud::Vision::ImageAnnotator.new
+    response_batch = image_annotator.document_text_detection image: image_path
+    scanned_text = ""
+    response_batch.responses.each do |res|
+      scanned_text << res.full_text_annotation.text
+    end
```

Comment:
Based on [this sample](https://cloud.google.com/vision/docs/ruby-client-migration#text_detection), but getting the `document_text` result rather than the "plain" `text` result.

I had tried the latter too but took a big accuracy hit.